store/activities.js

```diff
@@ -10,8 +10,8 @@ const config = [
     icon: 'document',
   },
   {
-    id: 'depositDates',
-    path: 'deposit-dates',
+    id: 'compliance',
+    path: 'compliance',
```

Comment:
_Compliance_ is not clear enough, it's too generic, compliance can be inspected comparingly to anything. I would keep it `deposit-compliance`.